### PR TITLE
bots: Add complete tests for offline testing of define bot.

### PR DESCRIPTION
**Pull request by abhijeetkaur**

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/define/fixtures/test_multi_type_word.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "request":{
+        "api_url":"https://owlbot.info/api/v1/dictionary/help?format=json"
+    },
+    "response":[
+        {
+            "type":"verb",
+            "defenition":"make it easier or possible for (someone) to do something by offering them one's services or resources.",
```
Comment (derAnfaenger) [inline zulip_bots/zulip_bots/bots/define/fixtures/test_multi_type_word.json]: Rather funny how the API misspelled "definition" :P

Comment (abhijeetkaur) [inline zulip_bots/zulip_bots/bots/define/fixtures/test_multi_type_word.json]: Yeah! I had to recheck that too! :sweat_smile:

**Comment by timabbott** [comment]
Nice, this looks great!  Ideally we wouldn't have to add these to the manifest, but that's an issue I think @eeshangarg is working on.  Merging.

**Comment by abhijeetkaur** [comment]
@timabbott I have more commits coming up, should I add them to Manifest file or not?

**Comment by eeshangarg** [comment]
@abhijeetkaur: Yes, add them to the Manifest file for now! :)

**Pull request merged**